features/poller/remote/src/main/java/org/opennms/netmgt/poller/remote/support/Log4j2StringAppender.java

```diff
@@ -29,6 +29,7 @@
 package org.opennms.netmgt.poller.remote.support;
 
 import java.io.ByteArrayOutputStream;
+import java.io.IOException;
```

Comment:
Something missing here ?

Reply:
oops, that's not actually necessary after all